spring-cloud-dataflow-admin-local/src/main/resources/admin.yml

```diff
@@ -18,7 +18,7 @@ spring:
     config:
       uri: http://localhost:8888
   datasource:
-    url: jdbc:h2:tcp://localhost:9092/mem:dataflow
+    url: jdbc:h2:tcp://localhost:9093/mem:dataflow
```

Comment:
This is still in the range that would be commonly used for a Kafka cluster. Maybe `19092`?